packages/editor-ui/src/components/InlineExpressionEditor/InlineExpressionEditorOutput.vue

```diff
@@ -34,13 +34,18 @@ const i18n = useI18n();
 const editor = ref<EditorView | null>(null);
 const root = ref<HTMLElement | null>(null);
 
+function addSegment(str: string, segment: Segment): string {
+	const text = segment.kind === 'resolvable' ? String(segment.resolved) : segment.plaintext;
+	return str.substring(0, segment.from) + text + str.substring(segment.to + 1);
+}
+
```

Comment:
Found this: https://share.cleanshot.com/87tnHllq

Reply:
And this: https://share.cleanshot.com/FgVjXwWW

Reply:
Good finds! should've tested better 😓 fixed now!